.travis.yml

```diff
@@ -353,3 +363,5 @@ jobs:
 branches:
   only:
     - master
+    # Delete next line before merge
+    - pb-db-testspec
```

Comment:
Is this before merge into master or merge into this branch?

Reply:
Removing with rebase to master